frontend/src/features/tasks/components/CompareResponsePanel.tsx

```
@@ -0,0 +1,260 @@
+  const { activeCompareGroup, selectResponse } = useCompareContext()
+
+  if (!activeCompareGroup) {
+    return null
+  }
+
+  const { compareGroupId, responses, allDone } = activeCompareGroup
+  const responseList = Array.from(responses.values())
+  const modelCount = responseList.length
+
+  // Calculate grid layout based on number of models
+  const gridCols = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)